posts/navarroaxel/13.running-graphics-apps-in-docker.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Running graphic apps in Docker: AWS WorkSpaces'
+published: true
+description: 'How to run GUI apps using X.Org and Docker in Linux or Windows'
+tags: 'tutorial, docker, aws, security'
+cover_image: ./assets/gui-apps-docker-cover.jpg
+id: 1253471
+---
+
+As a Linux user, I need to connect to an Amazon WorkSpace but the client app can only be run in specific old, but maintained, versions of Ubuntu. I tried with the latest Ubuntu version but it didn't work, so I've created a virtual machine with Ubuntu Focal 20.04 but this setup used 14GB of disk space 😵 only to open a window application. This didn't make any sense. Also this included issues when sharing the clipboard from the host to the VM and then to the virtual desktop hosted in Amazon. 🙄
+Then, I remembered that we can run graphic applications using Docker. Let's see how to connect a container to the [X Window System](https://en.wikipedia.org/wiki/X_Window_System) of the host.
+
+> Note: Treating option '--ncp-ciphers' as  '--data-ciphers' (renamed in OpenVPN 2.5).
```

Comment:
This part seems to be copy & paste from the previous article. Doesn't seem related to this one.